examples/player/simid_player.html

```diff
@@ -35,11 +35,39 @@
       simidPlayer.initializeAd();
     }
 
+    function toggleButtons() {
+      let adRequests = document.querySelectorAll(".ad_request");
+      adRequests.forEach(function(request) {
+        if (request.disabled == false) {
```

Comment:
You never need to compare things directly to boolean

`if (x == true)` can always be `if (x)`
`if (x == false)` -> `if (!x)`

or even better 
request.disabled = !request.disabled

or even better than that imo, would be to change this function to take in a param for if the buttons should be turned on or off, and set it to that value passed in. I think everytime you call this function, you have a specific desired result ..
Meaning for example, you call this function in skipAd. After skipAd is called, you always want to disable the buttons. Right now, this only works because you know that the skipAd button only shows when the button is active, but it would be a bit less brittle if in skipAd you called this function to specifically disable the buttons.  That way, you don't have to worry about the current state before calling this

Reply:
fixed toggle function to not depend on previous state